src/olympia/urls.py

```diff
@@ -140,7 +140,7 @@ def serve_javascript_catalog(request, locale, **kwargs):
             ),
             # Serve javascript catalog locales bundle directly from django
             re_path(
-                r'^static/js/i18n/(?P<locale>\w+)\.js$',
+                r'^static/js/i18n/(?P<locale>[a-z]{2}(?:-[A-Z]{2})?)\.js$',
```

Comment:
We need a broader regexp here: we currently support some 3 letters locales (`cak`, `dsb`, `fur`, `hsb`, `kab`...) and also `sr-Latn` in addons-server.

In nginx config we use `\w{2,3}(?:-\w{2,6})?`

Reply:
Updated.